.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+# Default Python for Travis is *some* Python 3.
+# Install dependencies.
+install:
+    - pip install -r requirements.txt
+    - pip install pycodestyle
+# Run tests.
+script:
+    - pycodestyle .
```

Comment:
not sure what the default behavior is but it seems like `pycodestyle` have 2 nice options we might want to use:

```
  --show-source        show source code for each error
  --show-pep8          show text of PEP 8 for each error (implies --first)
```